components/organisms/Repositories/repositories.tsx

```diff
@@ -130,7 +130,7 @@ const Repositories = ({ repositories }: RepositoriesProps): JSX.Element => {
               <TableTitle>Contributors</TableTitle>
             </div>
             <div className={clsx(classNames.cols.last30days, "hidden xl:flex")}>
-              <TableTitle>Last 30 Days</TableTitle>
+              <TableTitle>Last {range ?? 30} Days</TableTitle>
```

Comment:
refactor (blocking): use the range with the default value as mentioned up top.

```suggestion
              <TableTitle>Last {range} Days</TableTitle>
```